packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -1209,18 +1298,26 @@ export class IntervalCollection<TInterval extends ISerializableInterval>
         this.setSlideOnRemove(interval.end);
 
         if (newStart || newEnd) {
+            // In this case, where we change the start or end of an interval,
+            // it is necessary to remove and re-add the interval listeners.
+            // This ensures that the correct listeners are added to the ReferencePosition.
             this.localCollection.removeExistingInterval(interval);
+            this.localCollection.removeIntervalListeners(interval);
+
             if (newStart) {
                 const props = interval.start.properties;
+                this.client.removeLocalReferencePosition(interval.start);
```

Comment:
thanks for catching--I had meant to clean this up after noticing it while merging but forgot til just now :)